### PR TITLE
SDK-177: Support for fetching command properties

### PR DESCRIPTION
**Pull request by tanishgupta1**

### CODE REVIEW

File: bin/qds.py [hunk 1]
```diff
@@ -134,9 +136,18 @@ def runaction(cmdclass, args):
...
-    print(str(o))
+    if len(args) > 2:
+        sys.stderr.write("expecting not more than 2 arguments\n")
+        usage()
+
+    conn = Qubole.agent()
+    id = args.pop(0)
+    include_query_properties="false"
+    if len(args) == 1:
+        include_query_properties=args.pop(0)
```
Comment (harshshah87) [inline bin/qds.py]: Can you add validation for true/false here.

File: bin/qds.py [hunk 2]
```diff
@@ -238,6 +249,7 @@ def _create_cluster_info(arguments, api_version):
                                       aws_availability_zone=arguments.aws_availability_zone,
                                       vpc_id=arguments.vpc_id,
                                       subnet_id=arguments.subnet_id,
+                                      master_elastic_ip=arguments.master_elastic_ip,
```
Comment (harshshah87) [inline bin/qds.py]: Apart from the testing of include_query_properties, I am guessing everything else is already taken care by a different PR. Can you merge the latest branch?

File: bin/qds.py (hunk 1 above)
Comment (tanishgupta1) [inline bin/qds.py]: done

**Comment by harshshah87** [comment]
Merged.

**Pull request closed without merging**